bin/utils.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const walkSync = require( "walk-sync" );
+const fs = require( "fs" );
+
+function getFilesFromArgs( args ) {
+  let globs = args.slice();
+
+  if ( !globs.length ) {
+    globs.push( "test/*.js" );
```

Comment:
suggestion, maybe we should include: `test/**/*.js`, `*.jsx` and `.mjs` (node es modules filename, it's ugly, yes)